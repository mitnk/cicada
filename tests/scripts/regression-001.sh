export FOO=foo
echo "A${FOO}B$FOO/C${FOO}D"
echo "A$FOO/B${FOO}C$FOO/D"

# test cd/pwd on the absolute path string
mkdir -p foo1/bar1
cd foo1/bar1
cd ..
pwd | awk -F/ '{print $NF}'
cd ..
rm -rf ./foo1
echo =1=
yes | cat | cat | head -n 999999 | head -n 1
echo =2=
