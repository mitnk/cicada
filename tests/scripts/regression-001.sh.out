AfooBfoo/CfooD
Afoo/BfooCfoo/D
foo1
=1=
y
=2=
